src/models.py

```diff
@@ -17,25 +19,51 @@ def db_drop_and_create_all():
     db.drop_all()
     db.create_all()
 
```

Comment:
Please add a "comment" table with event id and user id foreign key.

Reply:
We can add this in a future PR. This will help to add comments from any user to the same event. Something like a chat for the event.

Reply:
Sounds good